aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -252,6 +253,15 @@ module aptos_framework::object {
         create_object_internal(owner_address, unique_address, false)
     }
 
+    /// Create an object at a specific address. Used by framework.
+    public(friend) fun create_object_at_address(
```

Comment:
This call is used basically for APT fungible asset metadata object to be created at 0x1. Maybe I can create a function call to manually create that at 0x1. Do you think it is worth the hassle?

Reply:
> This call is used basically for APT fungible asset metadata object to be created at 0x1.

@lightmark per https://github.com/aptos-labs/aptos-core/pull/12028 this functionality is offered via `aptos_coin::ensure_initialized_with_fa_metadata_for_test`

Reply:
@alnoki I mean creation...

Reply:
> @alnoki I mean creation...

@lightmark I'm confused by this statement, because earlier you mentioned:

> APT fungible asset metadata object to be created at 0x1

Which is what #12028 adds

Am I missing something?

Reply:
I mean this function is necessary to create an object at a specific address.
Now the only use case is to create APT FA at 0xA.(yeah, now we change it to 0xA). 
And your PR setup this in tests.

Reply:
@lightmark does this mean that `aptos_coin::ensure_initialized_with_fa_metadata_for_test()` should use `0xa` instead of `@aptos_framework`?

Reply:
@lightmark I've updated the links, but you replied to both and it looks good to me. Thanks for clarifying